numpy/typing/tests/data/pass/ma.py

```diff
@@ -16,7 +16,7 @@
 MAR_M_dt64: MaskedArray[np.datetime64] = np.ma.MaskedArray([np.datetime64(1, "D")])
 MAR_S: MaskedArray[np.bytes_] = np.ma.MaskedArray([b'foo'], dtype=np.bytes_)
 MAR_U: MaskedArray[np.str_] = np.ma.MaskedArray(['foo'], dtype=np.str_)
-MAR_T = cast(np.ma.MaskedArray[Any, np.dtypes.StringDType], np.ma.MaskedArray(["a"], "T"))
+MAR_T = cast(np.ma.MaskedArray[Any, np.dtypes.StringDType], np.ma.MaskedArray(["a"], dtype="T"))
```

Comment:
I happened to notice that this was passing "T" as the mask, so the resulting array had `dtype='U1'`. Perhaps not great that `MaskedArray` doesn't object if you pass `mask=some_string`!

Reply:
good catch!